docs/docs/guides/web3_config/index.md

```diff
@@ -9,7 +9,7 @@ sidebar_label: 'Web3 Config Guide'
 
 There is list of configuration params that can be set for modifying behavior of different functions in web3.js packages. Following is list of configuration options with details:
 
-- [handleRevert]( /api/web3-core/class/Web3Config#handleRevert) 
+- [handleRevert](/api/web3-core/class/Web3Config#handleRevert) 
```

Comment:
Instead of linking to the API docs here, I wonder if it would be better to link to the new section with the expanded information, and then link to the API docs from there.

Reply:
Good suggestion!